molecule/provisioner/ansible.py

```diff
@@ -202,11 +202,11 @@ class Ansible(base.Base):
     ::
 
         ANSIBLE_ROLES_PATH:
-          $ephemeral_directory/roles/:$project_directory/../
+          $ephemeral_directory/roles/:$project_directory/../:$HOME/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
```

Comment:
Using $HOME seems like a real bug to me. I am sure that $HOME is not defined on many platforms, do to assume it resolves.